codebuild/spec/buildspec_omnibus.yml

```diff
@@ -340,3 +340,16 @@ batch:
           TESTS: fuzz
           FUZZ_TIMEOUT_SEC: 60
 
+    - buildspec: codebuild/spec/buildspec_ubuntu_fuzz_artifacts.yml
```

Comment:
I'd prefer consistency in ordering for ease of reading: identifier, buildspec, env
Does it matter that it's alphabetical here?

Reply:
Fixed; order doesn't strictly mater here, but points for consistency.